### PR TITLE
PROJ1-4 Convert to Spring Boot

### PR DESCRIPTION
**Pull request by IkenoXamos**
Leave a clear description about what specific user stories were completed.

### CODE REVIEW

File: week4/notes/jira.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# JIRA
+
```
Comment (bjcady14) [inline week4/notes/jira.md:2]: There is a typo

**Review by bjcady14 — CHANGES_REQUESTED** [review]
Closing description!

File: week4/notes/jira.md (hunk 1 above)
Comment (IkenoXamos) [reply week4/notes/jira.md:2]: This has been fixed.

**Review by bjcady14 — APPROVED** [review]
Looks good

**Pull request merged**